scripts/autotracking.lua

```diff
@@ -295,35 +295,56 @@ function handleMoonstone(keyItem)
       moonstone.CurrentStage = 0
     end
   end
-  
+
 end
 
 --
 -- Handle items that can also show up in character's equipment slots.
--- This includes the Hero Medal and Robo's Ribbon. Masamune is handled
--- separately since it is more complicated.
+-- This includes the Hero Medal, Robo's Ribbon, and Grand Leon.
+-- Masamune is handled separately based on reforging with Melchior.
 --
 function handleEquippableItem(keyItem)
 
-  local equipmentSlot = AutoTracker:ReadU8(keyItem.address, 0)
-  local itemOwned = keyItem.found or equipmentSlot == keyItem.value
-  
-  local trackerItem = Tracker:FindObjectForCode(keyItem.name)  
-  if trackerItem and not trackerItem.Owner.ModifiedByUser then
-    trackerItem.Active = itemOwned
+  local trackerItem = Tracker:FindObjectForCode(keyItem.name)
+
+  if trackerItem.Owner.ModifiedByUser then
+    return
+  end
+
+  if not trackerItem then
+    trackerItem.Active = false
+    return
+  end
+
+  if keyItem.found then
+    trackerItem.Active = true
+    return
   end
 
+  -- to support Duplicate characters, need to check each character
+  local found = false
+  for pc=0,5 do
+    local address = 0x7E2650 + 0x50*pc + keyItem.offset
```

Comment:
This will miss Crono's equipment slots. Need to start at 7E2600 and extend the loop upper bound to 6.

Reply:
Whoops, great catch, thank you!